benchmark/_interfaces.py

```diff
@@ -87,3 +87,20 @@ def measure(f, *a, **kw):
         :return: Deferred firing when measurement has been taken, with the
             value of the measurement.
         """
+
+
+class IRequestScenarioSetup(Interface):
+    """
+    Setup for a load scenario.
+    It will provide a setup function and a make request function to
+    make requests of a certain type
+    """
+    def make_request():
+        """
+        Interface for request generator
+        """
+    def run_setup(self):
+        """
+        Interface for the scenario load setup
+        """
```

Comment:
I don't think I could implement this interface based on the documentation given here.  Can you elaborate on the lifecycle of this object in the class docstring and add information about what each of the methods is supposed to do and return to the method docstrings?

Also, `run_setup` has `self` as an argument and I suspect it shouldn't.